librna/rnalib.c

```diff
@@ -1073,16 +1074,41 @@ double mk_pf(double x) {
   return exp((-1.0 * x/100.0) / (GASCONST/1000 * (temperature + K0)));
 }
 
+double getScaleValue(int x) {
+  static bool init = false;
+  static const double mean_nrg = -0.1843;
```

Comment:
can we refactor the name to ALL CAPS since it is a constant?

Reply:
Sure, will do.